_posts/elements/2015-05-12-forms.md

```diff
@@ -23,29 +23,29 @@ info: Forms allow users to interact with the interface, set preferences, and ent
                     <p>Example: Jane Doe</p>
                 </div>
             </div>
             <div class="field">
-                <label class="field-label" for="email_address_default_01">Email address (default, disabled text field)</label>
-                <input class="field-input input-text" type="email" id="email_address_default_01" name="email_address_default_01" placeholder="e.g. jane@email.com"  aria-describedby="email_address_default_01_helper" disabled>
-                <div class="field-hint" id="email_address_default_01_helper">
+                <label class="field-label" for="email-address-default-01">Email address (default, disabled text field)</label>
+                <input class="field-input input-text" type="email" id="email-address-default-01" name="email-address-default-01" placeholder="e.g. jane@email.com"  aria-describedby="email-address-default-01-hint" disabled>
+                <div class="field-hint" id="email-address-default-01-hint">
                     <p>Example: jane@email.com</p>
                 </div>
             </div>
             <div class="field">
-                <label class="field-label field-required" for="email_address_default_helper">Email address (default, required text field with helper text)</label>
-                <input class="field-input input-text" type="email" id="email_address_default_helper" name="email_address_default_helper" placeholder="e.g. jane@email.com" aria-describedby="email_address_default_helper" required>
-                <div class="field-hint" id="email_address_default_helper">
+                <label class="field-label field-required" for="email-address-default-helper">Email address (default, required text field with hint text)</label>
+                <input class="field-input input-text" type="email" id="email-address-default-helper" name="email-address-default-hint" placeholder="e.g. jane@email.com" aria-describedby="email-address-default-hint" required>
+                <div class="field-hint" id="email-address-default-hint">
                     <p>Your email address will never be shared. We promise.</p>
                 </div>
             </div>
             <div class="field">
-                <label class="field-label field-required" for="email_address_default_err_helper">Email address (default, has error)</label>
-                <input class="field-input input-text has-error" type="email" id="email_address_default_err_helper" name="email_address_default_err_helper" placeholder="e.g. jane@email.com" value="jane@domain" aria-describedby="email_address_default_err_helper" required>
+                <label class="field-label field-required" for="email-address-default-err-hint">Email address (default, has error)</label>
+                <input class="field-input input-text has-error" type="email" id="email-address-default-err-hint" name="email-address-default-err-hint" placeholder="e.g. jane@email.com" value="jane@domain" required>
```

Comment:
Please do not abbreviate ``err``? Let's match the ``error`` label/term used elsewhere in the DOM.